oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/MonitoringCenterApiLiveTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.oneandone.rest.features;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import org.apache.jclouds.oneandone.rest.domain.MonitoringCenter;
+import org.apache.jclouds.oneandone.rest.domain.Server;
+import org.apache.jclouds.oneandone.rest.domain.Types;
+import org.apache.jclouds.oneandone.rest.domain.options.GenericDateQueryOptions;
+import org.apache.jclouds.oneandone.rest.domain.options.GenericQueryOptions;
+import org.apache.jclouds.oneandone.rest.internal.BaseOneAndOneLiveTest;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "MonitoringCenterApiLiveTest")
+public class MonitoringCenterApiLiveTest extends BaseOneAndOneLiveTest {
+
+   private Server currentServer;
+
+   private MonitoringCenterApi monitoringCenterApi() {
+
+      return api.monitoringCenterApi();
+   }
+
+   @BeforeClass
+   public void setupTest() throws InterruptedException {
+      currentServer = GetRandomServerWithMonitoringPolicy();
```

Comment:
This method can return null. It would be better to throw a `SkipException` here to avoid running the tests in this class if no server is available.
Would it make more sense to just create one server with the needed info, and runt he tests with that one?